lib/utils/getStdin.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @returns {Promise<string>}
+ */
+module.exports = async function getStdin() {
+	const chunks = [];
+
+	for await (const chunk of process.stdin) {
```

Comment:
[note] [`for await...of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for-await...of) has been available since Node.js v10.